staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -49,9 +49,8 @@ type AggregatorOptions struct {
 }
 
 // NewCommandStartAggregator provides a CLI handler for 'start master' command
-func NewCommandStartAggregator(out, err io.Writer, stopCh <-chan struct{}) *cobra.Command {
-	o := NewDefaultOptions(out, err)
-
+// with a default AggregatorOptions.
+func NewCommandStartAggregator(o *AggregatorOptions, stopCh <-chan struct{}) *cobra.Command {
```

Comment:
I still we should call them `defaults`. Makes it's purpose clearer.

Reply:
make sense